google/ggrc-core
# GGRC-1333/5375 Fix csv import with files with blank lines

# Issue description

Previously when we tried to import Assessment with evidence files with blank lines between them or Control with document files with blank lines between them we had the non-relevant error as well as didn't have the possibility to import the files.

# Steps to test the changes

1. Open import page
2. Assessment with evidence files with blank lines between them or Control with document files with blank lines between them
3. Assessment or Control will be imported

**Actual result:** the import fails.
**Expected result:** the empty lines are ignored. 

# Solution description

Empty line validation for preventing useless code execution and improving link parsing from the line.

# Sanity checklist


# PR Review checklist
